identity/app/controllers/editprofile/ConsentsJourney.scala

```diff
@@ -101,6 +104,17 @@ trait ConsentsJourney extends EditProfileControllerComponents {
       }
     }
 
+  private def newsletters() = {
```

Comment:
Returning no consents should be okay for now, unless we want to spend time implementing an error page. 

We're migrating away from this service and /consents/thank-you and /complete-consents will be moving to Gateway.

Reply:
Seems sensible - although I don't know what that translates to in terms of user experience so I don't think I could give an opinion on it either way. One for your team to decide!

Reply:
I've checked, the Identity team is happy with this approach